<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformManager</class>
 <widget class="QDialog" name="PlatformManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Platformbeheer</string>
  </property>
  <widget class="QListWidget" name="lstVersionOld">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Aanwezige versies:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butDelVersion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijder</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstVersionNew">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nieuwe/gewijzigde versies:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butGetNewVersions">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butAddVersion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Installeer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Beschikbare onderdelen:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ge√Ønstalleerde onderdelen:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstDeviceOld">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lstDeviceNew">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="butDelDevice">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butAddDevice">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Installeer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Platform:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Programmer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Platform:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Programmer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPlatformNew">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: blue;
font: italic 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtProgrNew">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: blue;
font: italic 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtProgrOld">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: blue;
font: italic 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPlatformOld">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: blue;
font: italic 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
